### PR TITLE
Puppet8

### PR DESCRIPTION
**Pull request by flepoutre**
Update code for Puppet8 compatibility.

### CODE REVIEW

File: manifests/hwtools.pp [hunk 1]
```diff
@@ -6,17 +6,18 @@
 # $dell_repo: use the dell repo for yumrepo, or a already defined one.
```
Comment (JGodin-C2C) [inline manifests/hwtools.pp]:
```suggestion

```

File: manifests/hwtools.pp [hunk 2]
```diff
@@ -6,17 +6,18 @@
...
 #  The yumrepo should have the name 'dell-omsa-indep'
 #
-class dell::hwtools(
+# @param dell_repo
```
Comment (JGodin-C2C) [inline manifests/hwtools.pp]: This documentation already exists. It is however not standardized.
 ```suggestion
# @param dell_repo use the dell repo for yumrepo, or a already defined one.
```

File: manifests/openmanage/redhat.pp [hunk 3]
```diff
@@ -6,10 +6,11 @@
 # $dell_repo: use the dell repo for yumrepo, or a already defined one.
 #  The yumrepo should have the name 'dell-system-update_dependent'
 #
-class dell::openmanage::redhat(
+# @param dell_repo
```
Comment (JGodin-C2C) [inline manifests/openmanage/redhat.pp:9]:
```suggestion
# @param dell_repo use the dell repo for yumrepo, or a already defined one. The yumrepo should have the name 'dell-system-update_dependent'
```

File: manifests/openmanage/redhat.pp [hunk 4]
```diff
@@ -6,10 +6,11 @@
 # $dell_repo: use the dell repo for yumrepo, or a already defined one.
```
Comment (JGodin-C2C) [inline manifests/openmanage/redhat.pp:6]:
```suggestion
```

File: manifests/hwtools.pp [hunk 5]
```diff
@@ -3,20 +3,18 @@
 #
 # Install hardware tools
 #
-# $dell_repo: use the dell repo for yumrepo, or a already defined one.
-#  The yumrepo should have the name 'dell-omsa-indep'
+# @param dell_repo use the dell repo for yumrepo, or a already defined one. The yumrepo should have the name 'dell-omsa-indep'.
```
Comment (flepoutre) [inline manifests/hwtools.pp:6]: Is it ok ?

File: manifests/hwtools.pp (hunk 2 above)
Comment (JGodin-C2C) [reply manifests/hwtools.pp]: It was already in the batch, please ignore.

File: manifests/hwtools.pp (hunk 1 above)
Comment (JGodin-C2C) [reply manifests/hwtools.pp]: It was already in the batch, please ignore.

**Pull request merged**